tests/integration/managed_array_tests.cpp

```diff
@@ -830,7 +830,9 @@ GPU_TEST(ManagedArray, PodTestGPU)
 
 TEST(ManagedArray, ExternalConstructorUnowned)
```

Comment:
@davidbeckingsale, @mdavis36, I can't get this test to pass.

Reply:
It also seems that using pinned memory for the CPU space conflicts with the ability to construct using a simple malloc'ed pointer (say from a 3rd party library).